pipelines/broad/rna_seq/RNAWithUMIsPipeline.wdl

```diff
@@ -19,9 +30,53 @@ workflow RNAWithUMIsPipeline {
 		File exonBedFile
 	}
 
+    if (defined(bam) && (defined(r1_fastq) || defined(r2_fastq))) {
+      call utils.ErrorWithMessage as ErrorMessageDoubleInput {
+        input:
+          message = "Bam and fastq files cannot both be defined as input"
+      }
+    }
+
+    if (!defined(bam) && (!defined(r1_fastq) && !defined(r2_fastq))) {
+      call utils.ErrorWithMessage as ErrorMessageMissingInput {
+         input:
+           message = "Either bam or fastqs must be defined."
+      }
+    }
+
+    if (defined(r1_fastq) || defined(r2_fastq)) {
+      if ((defined(r1_fastq) && !defined(r2_fastq)) || (defined(r2_fastq) && !defined(r1_fastq))) {
```

Comment:
```suggestion
      if !(defined(r1_fast1q) && defined(r2_fastq)) {
```
I think that should work?  If not (both are defined)

Reply:
ah yes, a much more succinct way of putting it. this should work the same. although i think wdl will yell unless i put parens around that whole thing: ` if (!(defined(r1_fastq) && defined(r2_fastq))) {`
ill try it out